src/EnergyPlus/SystemAvailabilityManager.hh

```diff
@@ -456,6 +459,14 @@ namespace SystemAvailabilityManager {
 		// manager is connected to air loop
 		bool SimHybridVentSysAvailMgr; // Set to false when a zone has two hybrid ventilation
 		// managers, one with air loop and one without
+		Real64 OperativeTemp; // Zone air operative temperature
+		Real64 CO2; // Zone air CO2 temperature
```

Comment:
This is not presently used, so delete.  Unless you add a new output for zone CO2 concentration, then it might be used.